keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -46,43 +43,57 @@
-light. There was significant discussion across issues, mailing list threads,
-and PRs that didn't lead to forward progress because it seemed we were trying
-to solve everything at once.
-
-Here we attempt to scope a single deliverable before moving on to discussing
-workflow, label states, and entry/exit criteria.
-
 ## Motivation
 
-At present, there are [2,155 open issues and 902 pull requests](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen) open in
+As of 09-08-2020, there are [3,004 open issues and 975 pull requests](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen) open in
 `kubernetes/kubernetes`.
 
-- [356 are categorized as `lifecycle/stale`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Fstale)
-- [246 are categorized as `lifecycle/rotten`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Frotten)
-- [690 are categorized as `lifecycle/frozen`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Ffrozen)
+- [197 are categorized as `lifecycle/stale`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Fstale)
+- [200 are categorized as `lifecycle/rotten`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Frotten)
+- [756 are categorized as `lifecycle/frozen`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Ffrozen)
 
-This makes for about 20% of open issues/PRs that are in some state of staleness.
+This makes for about 13% of open issues/PRs that are in some state of staleness.
 If we consider items marked as `lifecycle/frozen`, then we're looking at around
-42% of issues/PRs that could potentially require attention.
+38% of issues/PRs that could potentially require attention.
 
 ### Goals
 
-See [User Stories](#user-stories).
+Streamline the issue triage process used within the [kubernetes/kubernetes]
+repository.
 
-<<[UNRESOLVED]>>
+- Remove unused `triage` labels.
+- Create new workflow centered around `needs-triage` label auto-applied to new
+  issues.
 
-### Non-Goals
+See [User Stories](#user-stories) for more information.
 
-- Prescribing an issue triage workflow for all projects within Kubernetes orgs
 
-<<[/UNRESOLVED]>>
+### Non-Goals
+
+- Prescribing an issue triage workflow for all projects within Kubernetes orgs.
+- Requiring all community groups within Kubernetes to adhere to the issue triage
+  workflow introduced in this KEP.
 
 ## Proposal
 
+### New Workflow
+
+An additional [required label] called `needs-triage` will be applied automatically
+to issues created within the [kubernetes/kubernetes] repository similar to the
+current `needs-sig` or `needs-kind` labels. This serves as a boolean signal to
+community group members that the issue has not yet been triaged.
+
+After the issue has been triaged by a community group, an org member can apply
+the `triage/accepted` label using the `/triage accepted` bot command signaling
+that it is ready to be worked on.
```

Comment:
If we're signaling that an issue is being worked on, why aren't we using the existing `lifecycle/active`?
This workflow is already used in cluster lifecycle projects, kind, and sometimes test-infra.

Reply:
> If we're signaling that an issue is being worked on, why aren't we using the existing lifecycle/active?

Looks like we can improve the wording here. `triage/accepted` would mean that a SIG considers this issue as "being valid" and will take a look at it. It might not necessarily mean that the issue is currently being worked on (in that case, `lifecycle/active` should be applied to the issue).

@sftim's comment accurately describes the intent for the label - https://github.com/kubernetes/enhancements/pull/1930#discussion_r468880573